src/core/prerender/prerender.ts

```diff
@@ -161,6 +174,14 @@ export async function prerender(nitro: Nitro) {
       }
     }
 
+    // Check if file already exists, for example copied across from public assets
+    const fileAlreadyExists =
+      existsSync(join(nitro.options.output.publicDir, route.fileName!)) ||
+      existsSync(join(nitro.options.output.publicDir, route.route));
```

Comment:
We can optimize to reduce one stat if we check `fileName` ending with `/index.html`, then skip on parent existence (`manifest.json/index.html`)